backend/origuide/solver.py

```diff
@@ -97,6 +100,15 @@ def _set_forces(self):
         set_all_crease_forces(self.edges)
         set_all_face_forces(self.faces)
 
+    def _set_target_angles(self):
+        if self.edges_fold_angles is None:
+            for e in self.edges:
+                e.target_angle = angle_from_assignment(e.assignment)
```

Comment:
Maybe in case fold_angle for a given angle is `null` the property should be computed from the assignment (?)

Reply:
Hmm. Maybe.